### PR TITLE
Fix/cypress text casing match

### PR DESCRIPTION
**Pull request by tim-wilde-nimble**
Tech debt: renamed the base class for the cypress selector generation from "ProjectPage" to "Select" to both shorten the selectors a little and make their names a bit more descriptive (like `select-backlink`)

Small clean-up of the Cypress commands to remove a duplicated function to select the "save and continue button" - removed the old one that used the ID, and updated all affected buttons to have the `data-cy` attribute.

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/86340_TL-record-dates-LA-information/86856_comments-updated-correctly.js [hunk 1]
```diff
@@ -26,15 +26,15 @@ Cypress._.each(['iphone-x'], (viewport) => {
...
             });
         });
-    
+
         it('TC01: Navigates to comment section & type alphanumerical characters', () => {
             let alphanumeric = 'abcdefghijklmnopqrstuvwxyz 1234567890 !"£$%^&*(){}[]:@,./<>?~|'
             cy.get('[data-test="change-la-info-template-comments"]').click()
             cy.commentBoxLaInfo().type(alphanumeric)
-            cy.saveContinueBtn().click()
+            cy.saveAndContinueButton().click()//.click()
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/86340_TL-record-dates-LA-information/86856_comments-updated-correctly.js:34]: small comment left in which is probably not needed.

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/support/commands.js [hunk 2]
```diff
@@ -59,10 +59,10 @@ Cypress.Commands.add('selectSchoolListing', (listing) => {
     cy.saveLocalStorage()
 });
 
-// Save and Continue Button (Universal)
-Cypress.Commands.add('saveContinueBtn', () => {
-    cy.get('[id="save-and-continue-button"]')
-});
+// // Save and Continue Button (Universal)
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/support/commands.js:62]: Did you mean to leave this comment here?

**Comment by jordan-beard** [comment]
Did you run the cypress tests after making these changes?

**Review by Lisa-V-Sau — APPROVED** [review]
Besides the double slash which Jordan has commented on, all looks good!

**Pull request merged**